### ISSUE TITLE
When MissingShort is suppressed then param tag messages not emitted

### ISSUE THREAD

**Issue by phil-davis** [squizlabs/PHP_CodeSniffer#1989]
Suppress the requirement for a short description in the PHPdoc block, like:
```

	<rule ref="PEAR">
		<exclude name="Generic.Commenting.DocComment.MissingShort" />
	</rule>
```
then write a doc block like:
```
	/**
	 * 
	 * @param string $myParameter
	 * @return boolean
	 */
```
and it passes ``phpcs`` inspection.

But I expect ``phpcs`` to complain with a message like:
```
-----------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------------------
 56 | ERROR | Tag cannot be grouped with parameter tags in a doc comment (Generic.Commenting.DocComment.NonParamGroup)
-----------------------------------------------------------------------------------------------------------------------
```
If I do write a short description, e.g.:
```
	/**
	 * Short description of method
	 * 
	 * @param string $mailhogUrl
	 * @return ResponseInterface
	 */
```
then ``phpcs`` does complain about ``NonParamGroup``

Also note that when ``MissingShort`` is enabled, then ``phpcs`` complains about the missing short description (as expected), but it does not go on to complain about ``NonParamGroup`` (or other problems in the doc block).

I guess that the doc block parsing is getting terminated when no short description is found, and in that case is not going on to look for other problems.

**Comment by gsherwood** [comment]
Closed by PR #1990
